app/routes/entries/browse/$initialLetters/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { type LoaderArgs, redirect } from "@remix-run/server-runtime";
+
+export async function loader({ request, params }: LoaderArgs) {
```

Comment:
If we're not using the `request` param, we do not need to destructure it. This shows a (minor) warning in my editor:

<img width="794" alt="Screen Shot 2023-06-01 at 8 39 14 AM" src="https://github.com/plot-and-scatter/dchp3/assets/3538990/44b03d3e-003d-40c5-961d-909c94567a55">

So we can just write:

```ts
loader({ params })
```

Reply:
Done